lib/command/src/Obelisk/Command/Deploy.hs

```diff
@@ -72,6 +72,12 @@ deployInit thunkPtr deployDir sshKeyPath hostnames route adminEmail enableHttps
   --accidentally create a production deployment that uses development
   --credentials to connect to some resources.  This could result in, e.g.,
   --production data backed up to a dev environment.
+  putLog Notice "Writing default module.nix"
+  writeDeployConfig deployDir "module.nix"
+    "{ obeliskPkgs, ... }: {...}: {\n\
+    \  imports = [(obeliskPkgs.path + /nixos/modules/virtualisation/amazon-image.nix)];\n\
+    \  ec2.hvm = true;\n\
+    \}"
```

Comment:
Would rather not inline this... somehow piggy back on obelisk thunk's version of this... (see `mkBaseEc2`)